selfdrive/thermald/thermald.py

```diff
@@ -33,7 +33,7 @@
 CURRENT_TAU = 15.   # 15s time constant
 TEMP_TAU = 5.   # 5s time constant
 DISCONNECT_TIMEOUT = 5.  # wait 5 seconds before going offroad after disconnect so you get an alert
-PANDA_STATES_TIMEOUT = int(1000 * 1.5 * DT_TRML)  # 1.5x the expected pandaState frequency
+PANDA_STATES_TIMEOUT = int(100 * 1.5)  # 1.5x the expected pandaState frequency
```

Comment:
this was only correct because we were multiplying 1000ms by 0.5 which brought it correctly down to 500ms